cli/pkg/printer/resources.go

```diff
@@ -707,6 +707,50 @@ type billingIssue struct {
 	EventTime    string `header:"event_time,timestamp(ms|utc|human)" json:"event_time"`
 }
 
+func (p *Printer) PrintDeployments(deployments []*adminv1.Deployment) {
+	if len(deployments) == 0 {
+		p.PrintfWarn("No deployments found\n")
+		return
+	}
+
+	p.PrintData(toDeploymentsTable(deployments))
+}
+
+func toDeploymentsTable(deployments []*adminv1.Deployment) []*deployment {
+	res := make([]*deployment, 0, len(deployments))
+	for _, d := range deployments {
+		res = append(res, toDeploymentRow(d))
+	}
+	return res
+}
+
+func toDeploymentRow(d *adminv1.Deployment) *deployment {
+	var updatedOn string
+	if d.UpdatedOn != nil {
+		updatedOn = d.UpdatedOn.AsTime().Local().Format(time.DateTime)
+	}
+
+	return &deployment{
+		ID:            d.Id,
+		Environment:   d.Environment,
+		Branch:        d.Branch,
+		Status:        d.Status.String(),
+		RuntimeHost:   d.RuntimeHost,
+		StatusMessage: d.StatusMessage,
+		UpdatedOn:     updatedOn,
+	}
```

Comment:
Doesn't this get too wide for most terminals? A few suggestions:
- Remove the `ID`, `RuntimeHost` and `UpdatedOn` fields from the table, and instead have a `rill deployment show` command for getting the details for a specific deployment
- Do nice formatting for `Status` so it doesn't look like an enum
- Merge the `StatusMessage` field into `Status`, like when `StatusMessage` is not empty, `Status = fmt.Sprintf("%s: %s", status, statusMessage)`